packages/general-settings/components/GeneralSettings/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InstagramDirectUpload from '../InstagramDirectUpload/index';
```

Comment:
No need for the `/index` 😄 

```suggestion
import InstagramDirectUpload from '../InstagramDirectUpload';
```